src/main/java/sample/Application.java

```diff
@@ -0,0 +1,10 @@
+package sample;
+
+import org.springframework.boot.SpringApplication;
+
+public class Application {
```

Comment:
Since this will be a Spring Boot application, would you please add the `@SpringBootApplication` annotation to the class?